test/test3Spec.js

```diff
@@ -0,0 +1,29 @@
+xdescribe('SPEC 3', function() {
```

Comment:
What's this file? Why are we committing new xdescribe'd specs?

Reply:
This is a change that came with merging changes from the original repo into our fork. This I believe is a test to make sure there is proper behaviour for skipped tests.